docs/api.md

```diff
@@ -243,7 +243,11 @@ Synopsis:
 ```js
 articles.sync()
   .then(console.log.bind(console))
-  .catch(console.error.bind(console));
+  .catch(err => {
+    if (err.data === 401) {
```

Comment:
You can access the response through `err.response`; you should be able to get the status through `response.status`, which is probably clearer & more explicit than `data`, which is the json response body attached by to the response itself.